docs/en/api/route-object.md

```diff
@@ -10,15 +10,15 @@ The route object can be found in multiple places:
 
 - As the return value of calling `router.match(location)`;
 
-- Inside navigation guards as the first argument:
+- Inside navigation guards as the first and second argument:
```

Comment:
I think that is more a matter of preference, as I think both are correct. Since you're a vuejs member, I'll change it to your preference.